packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -217,12 +216,16 @@
     stroke: rgba(33, 36, 44, 0.32);
 }
 
-.MafsView .angle-arc,
-.MafsView .arc-right-angle {
+.MafsView .angle-arc-interactive {
     stroke: var(--mafs-blue);
     stroke-width: 0.1px;
 }
 
+.MafsView .angle-arc-static {
+    stroke: var(--static-gray);
+    stroke-width: 0.1px;
+}
+
```

Comment:
I could not for the life of me figure out how to pass the `stroke` value into the angle paths in the `angle-indictors.tsx` file, so I used CSS classes/styles for that.

Unfortunately, it means this is the only place where the way I handled the color is inconsistent with the rest.